cvat-canvas/src/typescript/masksHandler.ts

```diff
@@ -551,6 +588,10 @@ export class MasksHandlerImpl implements MasksHandler {
                 }
             }
 
+            if (drawData.brushTool) {
+                this.checkEmpty();
```

Comment:
checkEmpty uses ``this.tool``.
``this.tool`` initialized only in `this.updateBrushTools`
`this.updateBrushTools` not called in case of insertion. 

So, the condition is incorrect and probably this code should be inserted to `else` branch above

Reply:
Moved to `this.updateBrushTools`